Makefile

```diff
@@ -12,6 +12,9 @@ endif
 ifdef JOBS
 	export EXTRA_VARS += -e 'jobs=${JOBS}'
 endif
+ifdef LOCALHOST
+	export EXTRA_VARS += -c local
+endif
```

Comment:
Can we do this assignment in `deploy-jenkins.sh`?

Reply:
You might want to hold off on this for a short bit - it seems as if this way of doing `EXTRA_VARS` doesn't work. Hope to have a PR before long with an approach that does work.

Reply:
I think you might want to build on the new approach we adopt in https://github.com/alphagov/digitalmarketplace-jenkins/pull/409

Reply:
@bjgill OK I've tried to update it now to follow the new approach, please may you re-review?